crates/storage/src/connection/transaction.rs

```diff
@@ -784,12 +784,13 @@ pub(crate) mod dto {
         pub builtins: BuiltinCounters,
         pub n_steps: u64,
         pub n_memory_holes: u64,
-        pub data_availability: ExecutionDataAvailability,
+        pub data_availability: L1Gas,
+        pub total_gas_consumed: L1Gas,
```

Comment:
Oops -- we cannot just add this one. This is the storage DTO, so to keep compatibility with our serialized receipts in storage we need to add a new version of the struct(s)... :(

Reply:
good catch :bow: !